packages/web/lib/page.js

```diff
@@ -110,7 +110,7 @@ const getPageScope = (parent) => {
       });
       return null;
     },
-    config: (_) => (_) => async (ruleSet) => {
+    pageConfig: (_) => (_) => async (ruleSet) => {
```

Comment:
we use `config` in a few examples (like examples/web/thegazette.browse). Let's update them to use `pageConfig`